android/libraries/rib-base/src/main/java/com/badoo/ribs/core/Interactor.kt

```diff
@@ -90,7 +93,9 @@ abstract class Interactor<C : Parcelable, Content : C, Overlay : C, V : RibView>
 
     @CallSuper
     open fun onSaveInstanceState(outState: Bundle) {
-        outState.putString(KEY_TAG, tag)
+        val bundle = Bundle()
+        bundle.putString(KEY_TAG, tag)
+        outState.putBundle(BUNDLE_KEY, bundle)
```

Comment:
Why do we need to put `Bundle` inside `Bundle`?

Reply:
We don't. But so far it was also nested (check Node in dev), this is just keeping it that way by moving the responsibility of nesting it, since all main components now get the main bundle.